redux/_lib/findProjById.js

```diff
@@ -0,0 +1,3 @@
+const findProjectNumber = (id, projects) => projects.findIndex((elem) => elem.id === id);
```

Comment:
Do you think it deserves a separate function? If yes, please consider renaming it to `findProjectIndex`

Reply:
ok